packages/notebook/src/actions.tsx

```diff
@@ -678,7 +708,9 @@ export namespace NotebookActions {
    */
   export function runAllAbove(
     notebook: Notebook,
-    sessionContext?: ISessionContext
+    sessionContext: ISessionContext,
+    sessionDialogs: ISessionContextDialogs,
```

Comment:
Since the underlying `runCell` seems to be accepting `sessionDialogs` according to the current state of the PR (`sessionDialogs?`), maybe we could make it optional here as well?

Reply:
And in other `NotebookActions` functions like `runAll`, `runAndAdvance` and so on.